### PR TITLE
[PoC]: Sync documentation from ReVanced Docs to Docusaurus 

### PR DESCRIPTION
**Pull request by validcube**
This code is created for old Docusaurus repo, rewriting in progress

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  pull-requests: write
+
+jobs:
+  pull-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v4
+
+    - name: Pull docs from rv-docs repository
```
Comment (oSumAtrIX) [inline .github/workflows/docs.yml:19]: Don't abbreviate here

```suggestion
    - name: Pull documentation
```

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      run: | # cp -rfL docs/* ../docs fail? // cp -rf repositories/*/docs ../docs/* work but worse
+        git clone https://github.com/ReVanced/revanced-documentation rv-docs && cd rv-docs
+        git submodule update --init --recursive
```
Comment (oSumAtrIX) [inline .github/workflows/docs.yml:22]: Can't you use an action? Would be more elegant.

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    - name: Push changes to origin
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+        git add .
+        git commit -m "Updated docs"
+        git push origin master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (oSumAtrIX) [inline .github/workflows/docs.yml:41]: A commit action can be used.

File: .github/workflows/docs.yml (hunk 2 above)
Comment (validcube) [reply .github/workflows/docs.yml:22]: I prefer to write my first iteration of my CI with pure commands

action can be made at later iterations.

**Comment by Ushie** [comment]
The docusaurous branch is stale, as docusaurous may be separated, and a sync method hasn't been decided (there are some unsettled issues)

**Comment by oSumAtrIX** [comment]
I'd say we can allocate an issue for this?

**Comment by Ushie** [comment]
Sure, go ahead and open one as you were the one who raised the issue with this type of syncing

**Pull request closed without merging**